package.json

```diff
@@ -27,9 +27,9 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -P tslint-rules",
     "compile:script": "tsc -P script/tsconfig.json",
-    "lint": "yarn lint:src && yarn lint:prettier",
-    "lint:fix": "yarn lint:src:fix && yarn lint:prettier --fix",
-    "lint:prettier": "ts-node script/prettier.ts",
+    "lint": "yarn prettier && yarn lint:src",
+    "lint:fix": "yarn prettier --write && yarn lint:src:fix",
+    "prettier": "prettier --check \"./**/*.{yaml,yml}\" \"./app/**/*.{scss,ts,tsx,js,jsx,html}\" \"./script/**/*.{ts,js}\"",
```

Comment:
I'm wondering if this pattern can be simplified down to something like `**/*.{ext1,ext2,extN}` and the exclusion of dirs completely moved to `.prittierignore`. This should make it easier to reason about what's prettified and also makes it more straightforward to configure [git hooks](https://prettier.io/docs/en/precommit.html). 🤔

Reply:
i like that idea a lot @kachkaev! i'll take a stab at that :)